build_schema_ids.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
```

Comment:
1. Let's add a quick header explaining what this script does, please
2. Maybe let's have a `scripts` directory and put this (and any other scripts) there?